README.md

```diff
@@ -4,76 +4,86 @@ This repository contains Voluntary Product Accessibility Template&reg; (VPAT&reg
 
 The VPAT is a template used to document a product's conformance with accessibility standards and guidelines. The purpose of the VPAT is to assist customers and buyers in making preliminary assessments regarding the availability of commercial "Electronic and Information Technology", also referred to as "Information and Communication Technology" (ICT) products and services with features that support accessibility.
 
-> Documents in this repository will conform to VPAT&reg; `Version 1.0`, `Version 2.2`, `Version 2.3`, `Version 2.4` or the newer `Version 2.4 Rev`. **New documents in this repository should conform to the latest standard (2.4 Rev) using the [template provided](https://github.com/springernature/vpat/blob/master/base-template.md)** 
+## Creating a new VPAT
 
-### VPAT&reg; Version 1.0
+To create a new VPAT for a product, use a *copy* of the [base template](./base-template.md) as a starting point. Ensure that you update the following placeholders within your copied template:
 
-Voluntary Product Accessibility Template&reg; (VPAT&reg;) documents for Springer Nature products, outlining Section 508 accessibility information for Springer Nature products.
+- !INSERT PRODUCT!
+- !INSERT PRODUCT NAME!  
+- !INSERT PRODUCT DESCRIPTION!  
+- !INSERT DATE LAST EDITED!  
 
-The first table of the Template provides a summary view of the Section 508 Standards. The subsequent tables provide more detailed views of each subsection.
+then add your product to the list below.
```

Comment:
"product" or "product name"? (or a product link"?)

Reply:
I'll revisit this in a separate piece of work! 🙏